### ISSUE TITLE
[Bug] Partition Key Header not being saved to Outbox

### ISSUE THREAD

**Issue by iancooper** [BrighterCommand/Brighter#2601]
### Describe the bug
Whilst we explicitly save many headers, some have slipped through the net as we have added new ones. One of those is the PartitionKey, which needs adding into the headers (or send from the Outbox will fail)

### To Reproduce
Use an RDBMS Outbox with Kafka. Note that the Partition Key parameter is not stored

**Comment by garyfreer** [comment]
Hi @iancooper is this bug still being considered for being fixed? I am using a MySql Outbox and attempting to use PartitionKeys but they are not survive the process of writing/reading to/from the outbox table. Haven't yet been able to find a workaround.

Thanks!

**Comment by iancooper** [comment]
@garyfreer Yeah, work has been done. It will ship in the V10 release. We just need to fix up how we manage V9 and V10, particularly in docs, and then it will be at least in pre-release. Soon. 

(Tooling to upgrade your db for the new schema may be a little behind, hence the pre-release, but you can fix manually in that time window)

**Comment by siewching-lim** [comment]
Hi @iancooper, would it be possible for you to share the anticipated release date of V10? I would greatly appreciate it.

**Comment by iancooper** [comment]
@siewching-lim You can grab the pre-release with this fix now (10.0.0-preview.2). We anticipate a few other breaking changes for V10 (removing methods flagged obsolete in V9; better support for Cloud Events and OTel). Once we have those we will cut a final release. That might be mid-March by the time we have done everything and tested it.

10.0.0-preview.2 should be stable if you need the fix now.

**Comment by romtur** [comment]
Hi @iancooper. Is there any chance this could be fixed in v9 as well? We noticed that v10 is still in prerelease, with no new prereleases in over six months, while v9 continues to receive active updates.

**Comment by iancooper** [comment]
@romtur We hope to get V10 out by end of the year. I will have a look at the V9 backport. It depends if it makes a breaking change to the interface or not.

**Comment by iancooper** [comment]
I have re-opened the issue to look at V9

**Comment by iancooper** [comment]
Closing, as we are close to V10
